PowerEditor/src/Parameters.h

```diff
@@ -251,7 +251,7 @@ struct CmdLineParamsDTO
 	int _column2go = 0;
 	int _pos2go = 0;
 
-	LangType _langType;
+	LangType _langType = L_EXTERNAL;

```

Comment:
Why not L_NORMALTEXT ?

Reply:
because of this https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/Parameters.h#L223. So wanted to keep `struct CmdLineParamsDTO` aligned with `struct CmdLineParams`